Emby.Server.Implementations/TV/TVSeriesManager.cs

```diff
@@ -143,10 +143,33 @@ public IEnumerable<Episode> GetNextUpEpisodes(NextUpQuery request, User user, IE
             var allNextUp = seriesKeys
                 .Select(i => GetNextUp(i, currentUser, dtoOptions));
 
+            // allNextUp = allNextUp.OrderByDescending(i => i.Item1);
```

Comment:
Hmm?

Reply:
Was a full revert from https://github.com/jellyfin/jellyfin/pull/4595
If desired I can remove the commented line.

Reply:
Yes please